### ISSUE TITLE
Selectable rows should be indicated to the user

### ISSUE THREAD

**Issue by jouni** [vaadin/vaadin-grid#1005]
I think `<vaadin-grid-selection-column auto-select>` should add `cursor: pointer;` to the rows automatically, to indicate to the user that they can select the row by clicking.

**Comment by tomivirkki** [comment]
Does it make any sense to have the `autoSelect` property in `<vaadin-grid-selection-column>` to begin with. If it was a property of `<vaadin-grid>` then it could be reflected to attribute and a (default) theme could take care of stuff like this.

**Comment by jouni** [comment]
Does `autoSelect` do anything without the selection column?

**Comment by tomivirkki** [comment]
Yes I guess the end result could be the same as adding `<vaadin-grid-selection-column auto-select hidden></vaadin-grid-selection-column>` to a grid
